west.yml

```diff
@@ -50,7 +50,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: v2.4.0-ncs1
+      revision: v2.4.0-ncs-branch
```

Comment:
@ryanjh We don't put branch, but the current head of this branch.
So it should be: 38b88bdfa73ea632cb1cf34590c7d0c410fa6aa1

Reply:
thanks Joakim, I've fixed it.